test/exercism/user_test.rb

```diff
@@ -119,6 +119,27 @@ def test_increment_updates_single_record_per_user
     assert_equal 1, FiveADayCount.count
   end
 
+  def test_five_a_day_exercises_comments
+    fred = User.create(username: 'fred')
+    sarah = User.create(username: 'sarah')
+    jaclyn = User.create(username: 'jaclyn')
+    ACL.authorize(fred, Problem.new('ruby', 'bob'))
+
+    ex1 = UserExercise.create!(
+        user: sarah,
+        last_activity_at: 5.days.ago,
+        submissions: [Submission.create!(user: sarah, language: 'ruby', slug: 'bob', created_at: 22.days.ago)]
+    )
+    Comment.create!(submission: ex1.submissions.first, user: sarah, body: 'I like to comment')
+
+    UserExercise.create!(
+        user: sarah,
+        last_activity_at: 5.days.ago,
```

Comment:
This is great @bmulvihill. Thanks!
The build is now failing. I think it might be because we are using last_iteration_at instead of last_activity_at to pull the exercises.

I think I can merge this and make the changes.

Thanks!

cc: @sguermond @kytrinyx